modules/op-starting-pipelines.adoc

```diff
@@ -3,51 +3,53 @@
+. Optional: Modify the *Advanced Options* to add credentials for authenticating the specified private Git server or Docker registry.
 +
 .. Under *Advanced Options*, click *Show Credentials Options* and select *Add Secret*.
 .. In the *Create Source Secret* section, specify the following:
 
 ... A unique *Secret Name* for the secret.
 ... In the *Designated provider to be authenticated* section, specify the provider to be authenticated in the *Access to* field, and the base *Server URL*.
 ... Select the *Authentication Type* and provide the credentials:
-* For the *Authentication Type* `Image Registry Crendentials`, specify the *Registry Server Address* that you want to authenticate, and provide your credentials in the *Username*, *Password*, and *Email* fields.
+* For the *Authentication Type* `Image Registry Credentials`, specify the *Registry Server Address* that you want to authenticate, and provide your credentials in the *Username*, *Password*, and *Email* fields.
 +
 Select *Add Credentials* if you want to specify an additional *Registry Server Address*.
 
 * For the *Authentication Type* `Basic Authentication`, specify the values for the *UserName* and *Password or Token* fields.
-* For the *Authentication Type* `SSH Keys`, specify the value for the *SSH Private Key* field.
+* For the *Authentication Type* `SSH Keys`, specify the value of the *SSH Private Key* field.
 ... Select the check mark to add the secret.
 
 +
-You can add multiple secrets based upon the number of resources in your Pipeline.
+You can add multiple secrets based upon the number of resources in your pipeline.
 
-. Click *Start* to start the PipelineRun.
-. The *Pipeline Run Details* page displays the Pipeline being executed. After the Pipeline starts, the Tasks and Steps within each Task are executed.
+. Click *Start* to start the pipeline.
+. The *Pipeline Run Details* page displays the pipeline being executed. After the pipeline starts, the tasks and steps within each task are executed.
 You can:
-
-* Hover over the Tasks to see the time taken for the execution of each Step.
-* Click on a Task to see logs for each of the Steps in the Task.
-* Click the *Logs* tab to see the logs according to the execution sequence of the Tasks and use the *Download* button to download the logs to a text file.
+* Hover over the tasks to see the time taken to execute each step.
+* Click on a task to see logs for each of the steps in the task.
+* Click the *Logs* tab to see the logs according to the execution sequence of the tasks. You can also expand the pane and download the logs individually or in bulk, by using the relevant button.
+* Click the *Events* tab to see a stream of events generated by a pipeline run.
++
+You can use *Task Runs*, *Logs*, and *Events* tabs to assist in debugging a failed pipeline run or a failed task run.
 +
-.Pipeline run
-image::op_pipeline_run.png[]
+.Pipeline Run Details
```

Comment:
I am in two minds about whether this heading should be in sentence case or each word capitalised. If you are referring to the UI object itself then I can see why each word is capitalised. Otherwise, this heading could be in sentence case like all other headings in the docs library. My preference would be "Pipeline run details" in this case.

Reply:
It was indeed the UI element (the page) that I was referring to, but I want to follow the convention.